frontend/src/cards/ui/UnknownsAlert.tsx

```diff
@@ -47,9 +59,33 @@ function UnknownsAlert(props: {
           : breakdownVarDisplayName
       } was unknown.`;
 
+  const raceEthDiffMapText = `In cases where race and ethnicity are reported
+    separately, the map shows the higher of the two metrics.`;
+
   const percentageUnknown = unknowns[0][props.metricConfig.metricId];
 
-  return (
+  const diffRaceEthnicityText = raceEthnicityDiff
+    ? `This state reports race and ethnicity seperately.
```

Comment:
Typo, I think this is the one Allyson noticed in the demo maybe? "sepArately"

Reply:
thanks